### PR TITLE
Feature agregar automatizacion eliminar historia tipo feature

### PR DESCRIPTION
**Pull request by Jhen123**
image

### CODE REVIEW

File: features/support/hooks.js [hunk 1]
```diff
@@ -124,6 +124,32 @@ Before({ tags: "@createBugStory" }, async function(){
     this.createBugStoryHook = true;
 });
 
+Before({ tags: "@createFeatureStory" }, async function(){
+    console.log("Starting to create a feature story");
+    if (!this.createFeatureStoryHook){
+        //
```
Comment (shizukani9) [inline features/support/hooks.js]: borralo esto porfa

File: features/support/hooks.js [hunk 2]
```diff
@@ -124,6 +124,32 @@ Before({ tags: "@createBugStory" }, async function(){
     this.createBugStoryHook = true;
 });
 
+Before({ tags: "@createFeatureStory" }, async function(){
```
Comment (shizukani9) [inline features/support/hooks.js]: Se parece mucho al de @createBugStory .. quiza haya que refactorizar

File: features/support/hooks.js (hunk 1 above)
Comment (Jhen123) [reply features/support/hooks.js]: eliminado

File: features/support/hooks.js (hunk 2 above)
Comment (JorgeFloresChipana-QAV2) [reply features/support/hooks.js]: El hook de crear un bug lo puedes editar así para que también pueda crear feature story `Before({ tags: "@createBugStory or @createFeatureStory" }, async function(scenario){
    if (!this.createBugStoryHook){
        const addStoryButton = await DriverFactory.myDriver.wait(until.elementLocated(StoriesTab.addStoryButton));
        await DriverFactory.myDriver.wait(until.elementIsVisible(addStoryButton), configuration.browser.timeout);
        await addStoryButton.click();
        const storyTitleTextField = await DriverFactory.myDriver.wait(until.elementLocated(StoryPanel.storyTitleTextField));
        const storyTypeDropdown = await DriverFactory.myDriver.wait(until.elementLocated(StoryPanel.storyTypeDropdown));
        this.firstStoryName = RandomValues.alphanumeric(6);       
        await storyTitleTextField.sendKeys(this.firstStoryName);
        await storyTypeDropdown.click();
        const tags = scenario.pickle.tags;
        StoryPanel.locatorAux.value = StoryPanel.storyOptionInDropdown.value.replace("{0}", ((!!tags.find(tag => { return tag.name === '@createBugStory'; }) ? "bug":"feature")));
        const optionSelectedInDropdown = await DriverFactory.myDriver.wait(until.elementLocated(StoryPanel.locatorAux));
        await optionSelectedInDropdown.click();
        const saveButton = await DriverFactory.myDriver.wait(until.elementLocated(StoryPanel.saveButton));
        await saveButton.click();
        await DriverFactory.myDriver.wait(until.elementIsVisible(addStoryButton), configuration.browser.timeout);
        let storyItem = await DriverFactory.myDriver.wait(until.elementLocated(StoriesTab.previewStoryItemRow));
        storyItem = await DriverFactory.myDriver.wait(until.elementTextContains(storyItem, this.firstStoryBugName));
        await storyItem.click();
    }
});`

Comment (Jhen123) [reply features/support/hooks.js]: se realizo el cambio

**Comment by JorgeFloresChipana-QAV2** [comment]
sin observaciones

**Pull request merged**